app/controllers/tokens_controller.rb

```diff
@@ -28,4 +29,8 @@ def set_token
   def set_team_email
     ENV['TEAM_EMAIL'] || 'nomisapi@digital.justice.gov.uk'
   end
+
+  def set_parsed_permissions
```

Comment:
This could be a `before_action` filter called before the #new action.

Reply:
I considered that but, since it's only being used on one action (hence, no repetition to DRY out) and it's a non-standard instance var, I felt that it was better to be more explicit and set it inside the controller.

If you still think it's better to use a `before_action` filter I am happy to change it.

Reply:
No that's fine - it is only a one off use case so can leave it as it is.